src/mon/PGMap.cc

```diff
@@ -751,7 +751,6 @@ void PGMapDigest::dump_pool_stats_full(
     if (verbose) {
       tbl.define_column("QUOTA OBJECTS", TextTable::RIGHT, TextTable::RIGHT);
       tbl.define_column("QUOTA BYTES", TextTable::RIGHT, TextTable::RIGHT);
-      tbl.define_column("DIRTY", TextTable::RIGHT, TextTable::RIGHT);
       tbl.define_column("USED COMPR", TextTable::RIGHT, TextTable::RIGHT);
```

Comment:
Since cache tiering hasn't been officially deprecated, can we display this only when cache tiering is in use? Perhaps, we can add a check for `is_tier()`.

Reply:
@neha-ojha makes sense, updated: https://github.com/ceph/ceph/pull/40337/files#diff-63ebdbecacdafa4dbf5c269fb4954cf61e9e48bb05cc8373924a725f6226e2deR757-R759